eox_tagging/test/test_viewset.py

```diff
@@ -70,11 +71,31 @@ def test_create_tag(self):
             "target_id": "user_test",
             "owner_type": "user",  # default is site
             "access": "PRIVATE",
-            "expiration_date": "2020-12-04"
+            "expiration_date": "2020-12-04",
         }
+
         response = self.client.post("/api/v1/tags/", data, format='json')
+
         self.assertEqual(response.status_code, 201)
 
+    def test_create_tag_without_owner(self):
+        """"
+        Used to test creating a tag without an owner. The owner should be set as a default
+        to be the site.
+        """
+        data = {
+            "tag_type": "example_tag_2",
+            "tag_value": "example_tag_value",
+            "target_type": "user",
+            "target_id": "user_test",
+            "access": "PRIVATE",
+            "expiration_date": "2020-12-04",
+        }
+
+        response = self.client.post("/api/v1/tags/", data, format='json')
```

Comment:
could you use reverse to generate the url, this should be in the setup https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse